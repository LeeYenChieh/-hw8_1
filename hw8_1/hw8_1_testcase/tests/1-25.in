aocrmxOlHvlU1A1KU4yyleieq4Vpm NykVlSoaeTkm
tKnCVR7LK1jjG9WlkMojFO5dncdzKHGNaql8gIzDot7l7KqkgoE4OhH ecBlChbGicUkcLQk9hkpd9
L28X5rJT6FWbZnxJXfPqVofBo4cDZwF pSZHGEOAuNDhyoVdfOUF4FrzO3bo7y54a
5qCC1P43TxOKZTSqLhtrDAQQTvb2A49jf5fubvw2kZFhBihnvTRgPDRp6fBnJbj WwrXCaWMKoxXxQFxQjUWwODV9jxFfEYcRbopHR7xhSGfPBfG
J0NpP6Msc3Ez4dnRi76raCfZJwMRcn MsY5mBD5FuQrUtfnV8vXdJ2r4aJD7SvWt5bjPBDaND5EqVNzpFv5j1aYe2C
aocrmxOlHvlU1A1KU4yyleieq4Vpm add L28X5rJT6FWbZnxJXfPqVofBo4cDZwF add NeHUvdH1XgSZnCLlPxpdB8j68rTvSVM7tyEYpnbSYR8bfKy qvzvUU5fdQXFKYCWJ92UpsghtreA
5qCC1P43TxOKZTSqLhtrDAQQTvb2A49jf5fubvw2kZFhBihnvTRgPDRp6fBnJbj display
VlmX79ro1p1ojUjk9jNRPVC9yzyimEuWwFueGK2x6cCdOMpMQ92V RsY7Vp
