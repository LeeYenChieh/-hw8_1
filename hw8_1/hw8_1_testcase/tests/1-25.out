WwrXCaWMKoxXxQFxQjUWwODV9jxFfEYcRbopHR7xhSGfPBfG 
